lib/include/sha3iuf/sha3.h

```diff
@@ -59,7 +62,7 @@ void sha3_Init512(void *priv);
 
 enum SHA3_FLAGS sha3_SetFlags(void *priv, enum SHA3_FLAGS);
 
-void sha3_Update(void *priv, void const *bufIn, size_t len);
+void sha3_Update(void *priv, void const *bufIn, uint32_t len);
```

Comment:
This is incorrect. The size of the buffer in C and C++ is size_t. (You need to fix the calling code to comply with this). Specifically, we want to support buffers more than 2^32 bytes.

Reply:
Sure, I'll amend

Reply:
Done: [`c07148f` (#19)](https://github.com/brainhub/SHA3IUF/pull/19/commits/c07148ff483c2e205ec2b3903b4b31eb10f329a6)